xmodule/capa/capa_problem.py

```diff
@@ -495,6 +509,49 @@ def get_grade_from_current_answers(self, student_answers):
 
         return newcmap
 
+    def get_grade_from_answers(
+        self, student_answers: Optional[dict], correct_map: Optional[CorrectMap] = None
+    ) -> CorrectMap:
+        """
+        This method is based on `get_grade_from_current_answers` but it is used when
+        `ENABLE_GRADING_METHOD_IN_PROBLEMS` feature flag is enabled.
+
+        This method optionally recieve a `correct_map` to be used instead of the current one.
```

Comment:
Just a typo fix and a recommendation to elaborate on why this method is different.
```suggestion
        This method is based on `get_grade_from_current_answers` but it is used when
        `ENABLE_GRADING_METHOD_IN_PROBLEMS` feature flag is enabled.

        This method optionally receives a `correct_map` instead of the current one so ... 
```